kratos/python/add_vector_to_python.cpp

```diff
@@ -493,6 +494,47 @@ void  AddVectorToPython(pybind11::module& m)
     });
     py::implicitly_convertible<py::list, DenseVector<int>>();
 
+//***********************************************************************************
+    auto unsigned_int_vector_binder = CreateVectorInterface<DenseVector<unsigned int>>(m, "DenseVectorUnsignedInt");
```

Comment:
here we add the interface for DenseVector<unsigned int> which was missing from the python interface

Reply:
This could be done in an independent PR but ...

![image](https://user-images.githubusercontent.com/19991680/188814307-308bdec8-002f-4e4c-be9c-481b3eabefad.png)